float32 max_depth
int64 n_frame # samples current frame
int64 n_avg
int64 min
int64 max
float32 stddev
float32 mse # Mean square error
float32 rmse # Root mean square error
float32 mae # Mean absolute error
float32 me # Mean error
float32 stde # stddev of error
int64 count